website/source/guides/acl.html.markdown

```diff
@@ -0,0 +1,309 @@
+
+### Enable ACLs on Nomad Clients
+
+To enforce client endpoints, we need to enable ACLs on clients as well. This is simpler than servers, and we just need to set `enabled = true` in the [`acl` stanza](/docs/agent/configuration/acl.html). Once configured, we need to restart the client for the change.
+
+
+### Set an Anonymous Policy (Optional)
+
+The ACL system uses a whitelist or default-deny model. This means by default no permissions are granted.
+For clients making requests without ACL tokens, we may want to grant some basic level of access. This is done by setting rules
+on the special "anonymous" policy. This policy is applied to any requests made without a token.
+
+To permit anonymous users to read, we can setup the following policy:
+
+```text
+# Store our token secret ID
+$ export NOMAD_TOKEN="BOOTSTRAP_SECRET_ID"
+
+# Write out the payload
+$ cat > payload.json <<EOF
+{
+    "Name": "anonymous",
+    "Description": "Allow read-only access for anonymous requests",
+    "Rules": "
+        namespace \"default\" {
+            policy = \"read\"
+        }
+        agent {
+            policy = \"read\"
+        }
+        node {
+            policy = \"read\"
+        }
+    "
+}
+EOF
+
+# Install the policy
+$ curl --request POST \
+    --data @payload.json \
+    -H "X-Nomad-Token: $NOMAD_TOKEN" \
+    https://nomad.rocks/v1/acl/policy/anonymous
+
+# Verify anonymous request works
+$ curl https://nomad.rocks/v1/jobs
+```
+
+# Rule Specification
+
+A core part of the ACL system is the rule language which is used to describe the policy that must be enforced.
+We make use of the [HashiCorp Configuration Language (HCL)](https://github.com/hashicorp/hcl/) to specify rules.
+This language is human readable and interoperable with JSON making it easy to machine-generate. Policies can contain any number of rules.
+
+Policies typically have several dispositions:
+
+* `read`: allow the resource to be read but not modified
+* `write`: allow the resource to be read and modified
+* `deny`: do not allow the resource to be read or modified. Deny takes precedence when multiple policies are associated with a token.
+
+Specification in the HCL format looks like:
+
+```text
+# Allow read only access to the default namespace
+namespace "default" {
+    policy = "read"
+}
+
+# Allow writing to the `foo` namespace
+namespace "foo" {
+    policy = "write"
+}
+
+agent {
+    policy = "read"
+}
+node {
+    policy = "read"
+}
+```
+
+This is equivalent to the following JSON input:
+
+```json
+{
+    "namespace": {
+        "default": {
+            "policy": "read"
+        },
+        "foo": {
+            "policy": "write"
+        }
+    },
+    "agent": {
+        "policy": "read"
+    },
+    "node": {
+        "policy": "read"
+    }
+}
+```
+
+The [ACL Policy](/api/acl-policies.html) API allows either HCL or JSON to be used to define the content of the rules section.
+
+### Namespace Rules
```

Comment:
when reading this section, i still don't get what namespaces are supposed to mean, e.g. a `sensitive` namespace - how is that used / created / references? a job key? a client config key?